client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectSettings.tsx

```diff
@@ -15,10 +15,120 @@
+import { Button, Form } from "reactstrap";
+import { SuccessAlert } from "../../../../components/Alert.jsx";
+import { Loader } from "../../../../components/Loader.tsx";
+import { RtkErrorAlert } from "../../../../components/errors/RtkErrorAlert.tsx";
+import { Project } from "../../../projectsV2/api/projectV2.api.ts";
+import { usePatchProjectsByProjectIdMutation } from "../../../projectsV2/api/projectV2.enhanced-api.ts";
+import ProjectDescriptionFormField from "../../../projectsV2/fields/ProjectDescriptionFormField.tsx";
+import ProjectNameFormField from "../../../projectsV2/fields/ProjectNameFormField.tsx";
+import ProjectNamespaceFormField from "../../../projectsV2/fields/ProjectNamespaceFormField.tsx";
+import ProjectVisibilityFormField from "../../../projectsV2/fields/ProjectVisibilityFormField.tsx";
+import { ProjectV2Metadata } from "../../../projectsV2/show/ProjectV2EditForm.tsx";
+import ProjectPageDelete from "./ProjectDelete.tsx";
+interface ProjectSettingsFormProps {
+  project: Project;
+}
+export function ProjectSettingsForm({ project }: ProjectSettingsFormProps) {
+  const {
+    control,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<ProjectV2Metadata>({
+    defaultValues: {
+      description: project?.description,
+      name: project.name,
+      namespace: project.namespace,
+      visibility: project.visibility,
+    },
+  });
+
+  const [updateProject, { isLoading, error, isSuccess }] =
+    usePatchProjectsByProjectIdMutation();
+
+  const isUpdating = isLoading;
+
+  const onSubmit = useCallback(
+    (data: ProjectV2Metadata) => {
+      updateProject({
+        "If-Match": project.etag ? project.etag : undefined,
+        projectId: project.id,
+        projectPatch: data,
+      });
+    },
+    [project, updateProject]
+  );
+
+  return (
+    <div>
+      {error && <RtkErrorAlert error={error} />}
+      {isSuccess && (
+        <SuccessAlert dismissible={false} timeout={0}>
+          <p className="p-0">The project has been successfully updated.</p>
+        </SuccessAlert>
+      )}
+
+      <Form
+        className="form-rk-green"
+        noValidate
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <ProjectNameFormField name="name" control={control} errors={errors} />
+        <ProjectNamespaceFormField
+          name="namespace"
+          control={control}
+          entityName="project"
+          errors={errors}
+        />
+        <ProjectDescriptionFormField
+          name="description"
+          control={control}
+          errors={errors}
+        />
+        <ProjectVisibilityFormField
+          name="visibility"
+          control={control}
+          errors={errors}
+        />
+        <div className={cx("d-flex", "justify-content-end")}>
+          <Button disabled={isUpdating} className="me-1" type="submit">
+            {isUpdating ? (
+              <Loader inline={true} size={16} />
+            ) : (
+              <ArrowClockwise size={16} className="me-1" />
```

Comment:
The icon is not aligned with the text:

```suggestion
              <ArrowClockwise size={16} className={cx("bi", "me-1")} />
```